src/index.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import changeCase from 'case';
 import styleParser from 'style-attr';
 
+function isFunction(thing) {
```

Comment:
You can just do `thing.constructor === Function` or `typeof thing === 'function'`.